app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -176,25 +176,29 @@ def parse_openshift_image(openshift_image)
         ref = ContainerImage::DOCKER_PULLABLE_PREFIX + id
```

Comment:
@zeari I am sorry but on second thought maybe this is not enough also. without id and ref there is not much meaning to `parser_container_image`. Maybe we should also try to get the name from the "regular" metadata, which also might be empty.
```
id = openshift_image[:dockerImageReference] || openshift_image[:metadata][:name] || ""
```
also anyway the field name should start with a lowercase d IIUC